lib/config/validation.ts

```diff
@@ -97,7 +98,8 @@ export function getParentName(parentPath: string | undefined): string {
 export async function validateConfig(
   config: RenovateConfig,
   isPreset?: boolean,
-  parentPath?: string
+  parentPath?: string,
+  isGlobalConfig?: boolean
```

Comment:
```suggestion
  isGlobalConfig: boolean
```

make mandatory to not messup migration